redash/models/users.py

```diff
@@ -187,8 +187,14 @@ def get_by_api_key_and_org(cls, api_key, org):
         return cls.get_by_org(org).filter(cls.api_key == api_key).one()
 
     @classmethod
-    def all(cls, org):
-        return cls.get_by_org(org).filter(cls.disabled_at.is_(None))
+    def all(cls, org, is_invitation_pending=None):
```

Comment:
Some delicate stuff here. Could we avoid having this code by providing better hints from the frontend? If not, can we add a test for this?

Reply:
Of course I'll add tests for this. I'll try to explain the idea behind this, maybe I'm doing something wrong. We have a method to get all disabled users (`all_disabled`) and this one - to get all enabled users. But in some cases we need all enabled users, and sometimes - only active or only pending. So I added parameter `is_invitation_pending` which is a filter: 

- `is_invitation_pending=None` - fetch all
- `is_invitation_pending=False` - only active
- `is_invitation_pending=True` - only pending.

I don't know what is the right way to create API like this; this implementation works as I need, so ATM we have what we have. But I'll appreciate any suggestions for improving this.

Reply:
Sorry, I'm a total SQLAlchemy n00b, but I'd expect something similar to an ActiveRecord scope to be used here.

Reply:
At first it seemed like a terrible API design, but on second thought maybe it's ok...

Reply:
Surely there must be a way to achieve something like `User.all.pending`?

Reply:
@rauchy Yes, that's another possible option. Therefore I need your help to choose better one